src/main/java/net/snowflake/ingest/connection/RequestBuilder.java

```diff
@@ -177,7 +204,7 @@ public RequestBuilder(
       String schemeName,
       String hostName,
       int portNum) {
-    this(accountName, userName, keyPair, schemeName, hostName, portNum, null);
+    this(accountName, userName, keyPair, schemeName, hostName, portNum, null, null, null);
```

Comment:
There's a lot of `null`s being given to this constructor, does it make sense to have it be its own specialized constructor? Might reduce the amount of code changed in this PR.